src/lib/isMobilePhone.js

```diff
@@ -17,6 +17,7 @@ const phones = {
   'ar-SA': /^(!?(\+?966)|0)?5\d{8}$/,
   'ar-SY': /^(!?(\+?963)|0)?9\d{8}$/,
   'ar-TN': /^(\+?216)?[2459]\d{7}$/,
+  'ar-QA': /^(\+?974)?(\d{8}(\,\d{8}){0,2})$/,
```

Comment:
I'm not sure if `,` is allowed here and there is some carrier prefixes for Qatari mobile phones. You can find the full list [here](https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000AB0001PDFE.pdf)

Reply:
I missed that, thanks for the catch.

Reply:
Ok I check the regex, thanks for your comment!